### ISSUE TITLE
Size grip on the main window

### ISSUE THREAD

**Issue by mika76** [atom/status-bar#181]
I think a sizing grip on the status bar would be great. I'm not sure where this great usability feature has disappeared lately with apps, but it's so much easier to grip a block then a corner...

Atom version: 1.12.7

**Comment by thomasjo** [comment]
Thanks for the suggestion!

First of all, this cannot really be resolved via status-bar (which can also be disabled, thus rendering this functionality void). Second of all, the "grip" has been removed from most apps (and OS-level UI guidelines) since it's a false affordance to some extent. Application windows can be resized from any edge, not only the traditional lower right corner. Having a grip in the lower right gives conveys an affordance of the UI only being resizable from there, whereas in reality it can be resized from any of the edges. Therefore, it makes much more sense to remove the "grip" to avoid telling a partial truth.

**TL; DR** What you're requesting does not really make sense from a UX perspective in a modern GUI, where every edge can be used to resize the window.

**Comment by mika76** [comment]
Thanks @thomasjo. Is this possible to do as an addin? If so could you point me in the direction of where I could get started doing one?

**Comment by thomasjo** [comment]
Since this is not something that is implemented in Electron (as far as I know), you'll likely have to interact directly with the OS-level window handle via [`BrowserWindow.getNativeWindowHandle()`](http://electron.atom.io/docs/api/browser-window/#wingetnativewindowhandle). That means you need to handle support for Windows, macOS and Linux (which likely means multiple window managers) yourself. It might also involve needing to write a native Node.js module.

Sadly, I don't have any specifics for you, since I've never attempted to do anything like this with Electron.
